addons/medical_statemachine/stringtable.xml

```diff
@@ -4,33 +4,43 @@
         <Container name="Settings">
             <Key ID="STR_ACE_Medical_statemachine_subCategory">
                 <English>States</English>
+                <German>Zustände</German>
             </Key>
             <Key ID="STR_ACE_Medical_statemachine_fatalInjuryCondition_DisplayName">
                 <English>Fatal Injury Player</English>
+                <German>Tötliche Spielerverletzungen</German>
             </Key>
             <Key ID="STR_ACE_Medical_statemachine_fatalInjuryCondition_Description">
                 <English>Defines when Player can receive a fatal injury</English>
+                <German>Definiert wann ein Spieler tötliche Verletzungen bekommen kann.</German>
             </Key>
             <Key ID="STR_ACE_Medical_statemachine_fatalInjuryConditionAI_DisplayName">
                 <English>Instant death AI</English>
+                <German>Direkter Tot KI</German>
             </Key>
             <Key ID="STR_ACE_Medical_statemachine_fatalInjuryConditionAI_Description">
                 <English>Defines if AI will be able to die instantly</English>
+                <German>Legt fest, ob eine KI direkt sterben kann</German>
             </Key>
             <Key ID="STR_ACE_Medical_statemachine_unconsciousConditionAI_DisplayName">
                 <English>Forbid unconscious AI</English>
+                <German>Verhindere KI-Beuwsstlosigkeit</German>
             </Key>
             <Key ID="STR_ACE_Medical_statemachine_unconsciousConditionAI_Description">
                 <English>Defines if AI will be denied to go unconscious and will die instead</English>
+                <German>Soll eine KI bewusstlos werden oder direkt sterben?</German>
```

Comment:
English version is not a question.
Last time you translated "Defines if" with "Legt fest, ob"
Why change to a different style here?